doc/classes/File.xml

```diff
@@ -21,6 +21,7 @@
 		    return content
 		[/codeblock]
 		In the example above, the file will be saved in the user data folder as specified in the [url=https://docs.godotengine.org/en/latest/tutorials/io/data_paths.html]Data paths[/url] documentation.
+		[b]Note:[/b] if you want to access resources that ship with your game, you need to use [ResourceLoader]. Because of how assets are imported [File] will work in the editor but break in an exported project.
```

Comment:
```suggestion
		[b]Note:[/b] To access project resources once exported, it is recommended to use [ResourceLoader] instead of the [File] API, as some files are converted to engine-specific formats and their original source files might not be present in the exported PCK package.
```